docs-src/docs/guides/walkthrough.md

```diff
@@ -0,0 +1,317 @@
+
+A load balancer also has a notion of a health check - a way to assess whether or not the service is healthy and running properly. By default, Layer0 configures the health check of a load balancer based upon a simple TCP ping to port 80 every thirty seconds. Also by default, this ping will timeout after five seconds of no response from the service, and two consecutive successes or failures are required for the service to be considered healthy or unhealthy.
+
+Here, we'll create a new load balancer named **guestbook-lb** inside of our environment named **demo-env**. The load balancer will listen on port 80, and forward that traffic along to port 80 in the Docker container using the HTTP protocol. Since the port configuration is already aligned with the default health check, we don't need to specify any health check configuration when we create this load balancer. At the command prompt, execute the following:
+
+`l0 loadbalancer create --port 80:80/http demo-env guestbook-lb`
+
+This will give us the following output:
+
+```
+LOADBALANCER ID  LOADBALANCER NAME  ENVIRONMENT  SERVICE  PORTS       PUBLIC  URL
+guestbodb65a     guestbook-lb       demo-env              80:80/HTTP  true
+```
+
+The following is a summary of the arguments passed in the above command:
+
+* `loadbalancer create`: creates a new load balancer
+* `--port 80:80/HTTP`: instructs the load balancer to forward requests from port 80 on the server to port 80 in the Docker container using the HTTP protocol
+* `demo-env`: the name of the environment in which you are creating the load balancer
+* `guestbook-lb`: a name for the load balancer itself
+
+---
+
+### Part 3: Deploy the Docker Task Definition
+
+The `deploy` command is used to specify the Docker task definition that refers to a web application.
+
+Here, we'll create a new deploy called **guestbook-dep** that refers to the **Guestbook.Dockerrun.aws.json** file you obtained earlier. At the command prompt, execute the following:
+
+`l0 deploy create Guestbook.Dockerrun.aws.json guestbook-dep`
+
+This will give us the following output:
+
+```
+DEPLOY ID        DEPLOY NAME    VERSION
+guestbook-dep.1  guestbook-dep  1
+```
+
+The following is a summary of the arguments passed in the above command:
+
+* `deploy create`: creates a new deployment and allows you to specify a Docker task definition
+* `Guestbook.Dockerrun.aws.json`: the file name of the Docker task definition (use the full path of the file if it is not in your current working directory)
+* `guestbook-dep`: a name for the deploy, which you will use later when you create the service
+
+_The Deploy Name and Version are combined to create a unique identifier for a deploy. If you create additional deploys named **guestbook-dep**, they will be assigned different version numbers._
```

Comment:
... "you can always specify the latest version when targeting a deploy by using `<deploy name>:latest`, e.g. `guestbook-dep:latest`"